web/src/components/netflow-overview/panel-kebab.tsx

```diff
@@ -58,6 +60,23 @@ export const PanelKebab: React.FC<PanelKebabProps> = ({ id, options, setOptions
     [setOptions, options]
   );
 
+  const onOverviewExport = React.useCallback(() => {
+    const overview_flex = document.getElementById(id);
```

Comment:
nipticking here but I see there is a margin on right + bottom in the exported screen

![overview](https://user-images.githubusercontent.com/91894519/235852302-a505c680-bb8f-4e71-82ac-b72a5b3ef54d.png)


```suggestion
    const overview_flex = document.getElementById(id)?.children[0] as HTMLElement | undefined;
```
taking the first child would remove it

![overview without margin](https://user-images.githubusercontent.com/91894519/235852427-b44347ab-0150-4852-b913-b8f7029dac45.png)